WooCommerce/Classes/ViewRelated/Products/Edit Product/BottomSheetListSelector/BottomSheetProductType.swift

```diff
@@ -77,13 +76,13 @@ public enum BottomSheetProductType: Hashable, Identifiable {
         case .simple(let isVirtual):
             if isVirtual {
                 return NSLocalizedString(
-                    "bottomSheetProductType.simpleVirtual.title",
-                    value: "Simple virtual product",
+                    "bottomSheetProductType.simpleVirtualProduct.title",
```

Comment:
Have to also change the key for the string value change to apply. Not completely sure why, but without doing this, the string change doesn't appear in the simulator.